src/main/core/Listener/AuthenticationFailureListener.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Claroline\CoreBundle\Listener;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Security\Core\Event\AuthenticationFailureEvent;
+
+class AuthenticationFailureListener
```

Comment:
IMO you can reuse `Claroline\CoreBundle\Listener\AuthenticationFailureHandler`.

Reply:
Sorry for the late answer and thanks you :)